src/main/resources/db/migration/V4_0__move_customer_to_stakeholder.sql

```diff
@@ -0,0 +1,41 @@
+-- Create new tables
+
+create table stakeholder (
+   id bigint not null auto_increment,
+    address varchar(255),
+    care_of varchar(255),
+    country varchar(255),
+    first_name varchar(255),
+    last_name varchar(255),
+    stakeholder_id varchar(255),
+    type varchar(255),
+    zip_code varchar(255),
+    errand_id varchar(255) not null,
+    primary key (id)
+) engine=InnoDB;
+
+create table contact_channel (
+   stakeholder_id bigint not null,
+    type varchar(255),
+    value varchar(255)
+) engine=InnoDB;
+
+alter table contact_channel
+   add constraint fk_stakeholder_contact_channel_stakeholder_id
+   foreign key (stakeholder_id)
+   references stakeholder (id);
+
+alter table stakeholder
+  add constraint fk_errand_stakeholder_errand_id
+  foreign key (errand_id)
+  references errand (id);
+
+-- Migrate data
+
+insert into stakeholder(errand_id, type, stakeholder_id)
+select id, customer_type, customer_id from errand;
+
+-- Update errand
+
```

Comment:
reflect all changes in src/test/db/scripts/schema.sql in this migration-script (removed indexes, constraint, etc)